data/data/aws/vpc/vpc.tf

```diff
@@ -1,6 +1,6 @@
 locals {
-  new_worker_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,1)}"
-  new_master_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,0)}"
+  new_compute_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,1)}"
+  new_controlplane_cidr_range = "${cidrsubnet(data.aws_vpc.cluster_vpc.cidr_block,1,0)}"
```

Comment:
`controlplane` -> `control_plane`?

Reply:
Doing my best to make this swap throughout the changes with the machine pool name being `"control_plane"`. Some things like AWS role names will need to be hyphenated so I'll be combing over those.